cli/initiator/initiator.go

```diff
@@ -30,76 +28,76 @@ var StartDKG = &cobra.Command{
 		██║  ██║██╔═██╗ ██║   ██║    ██║██║╚██╗██║██║   ██║   ██║██╔══██║   ██║   ██║   ██║██╔══██╗
 		██████╔╝██║  ██╗╚██████╔╝    ██║██║ ╚████║██║   ██║   ██║██║  ██║   ██║   ╚██████╔╝██║  ██║
 		╚═════╝ ╚═╝  ╚═╝ ╚═════╝     ╚═╝╚═╝  ╚═══╝╚═╝   ╚═╝   ╚═╝╚═╝  ╚═╝   ╚═╝    ╚═════╝ ╚═╝  ╚═╝`)
-		if err := cli_utils.SetViperConfig(cmd); err != nil {
-			return err
-		}
-		if err := cli_utils.BindInitFlags(cmd); err != nil {
-			return err
-		}
 		logger, err := cli_utils.SetGlobalLogger(cmd, "dkg-initiator")
```

Comment:
This fails because it tries to open LogFilePath flag but it has not yet been populated (its populated just below in BindInitFlag)

Reply:
fixed